x-pack/test/ui_capabilities/common/fixtures/plugins/foo_plugin/index.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function (kibana) {
+  return new kibana.Plugin({
+    require: ['kibana', 'elasticsearch', 'xpack_main'],
+    name: 'foo',
+    uiExports: {
+      app: {
+        title: 'Foo',
+        order: 1000,
+        euiIconType: 'uiArray',
+        description: 'Foo app',
+        main: 'plugins/foo_plugin/app',
+      },
+    },
+
+    config() {},
+
+    init(server) {
+      server.plugins.xpack_main.registerFeature({
+        id: 'foo',
+        name: 'Foo',
+        icon: 'upArrow',
+        navLinkId: 'foo_plugin',
+        app: ['kibana'],
+        catalogue: ['foo'],
+        privileges: {
+          all: {
+            savedObject: {
+              all: ['foo'],
+              read: ['index-pattern', 'config'],
+            },
+            ui: ['create', 'edit', 'delete', 'show'],
+          },
+          read: {
+            savedObject: {
+              all: [],
+              read: ['foo', 'config'],
```

Comment:
__question:__ just for understanding/curiosity: why `read.savedObject.read` doesn't include `index-pattern`?

Reply:
No real reason, I do agree that it'd make it more consistent with the other actual uses to have it here, I'll go ahead and add it.